Scripts/incrementMinorVersion.ps1

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+<#
+.Synopsis
+    Increment the minor version string in the gradle.properties if the major,
+    minor, or patch version hasn't been manually updated.
+.Description
+    Assumptions:
+        Targets Gradle.properties
+        This script assumes it is run from the repo root.
+        Minor version is typically auto-incremented.
+
+#>
+
+function Update-ReadmeVersion([string]$readmeFilePath, [string]$version) {
+	$readmeFileContent = Get-Content -Path $readmeFilePath -Raw
+	$readmeFileContent = $readmeFileContent -replace "\d{1,}\.\d{1,}\.\d{1,}", $version
+	Set-Content -Path $readmeFilePath $readmeFileContent
+}
+
+function Update-TelemetryVersion([string]$telemetryFilePath, [string]$version) {
+	$telemetryFileContent = Get-Content -Path $telemetryFilePath -Raw
+	$telemetryFileContent = $telemetryFileContent -replace "\d{1,}\.\d{1,}\.\d{1,}", $version
+	Set-Content -Path $telemetryFilePath $telemetryFileContent
+}
+
+function Update-PackageVersion([string]$propertiesFilePath, [string]$version) {
```

Comment:
consider using cast into version like `[version]$version`. That will give you a strongly typed version object with Major, Minor, Build fields.

Reply:
thanks! I had completely forgotten about that type for some reason... Fixed.